.dockerignore

```diff
@@ -0,0 +1,2 @@
+.tox
+notebooks
```

Comment:
Yes, but not included in the docker builds. The notebooks are for people to graph when jvmquake would step in given a gclog (so they can simulate if it would kick in on their apps).